command: python -m pytest test_issue4.py -v
result:
================================================================================================ test session starts ================================================================================================
platform win32 -- Python 3.10.6, pytest-7.4.3, pluggy-1.3.0 -- C:\Users\Артём\AppData\Local\Programs\Python\Python310\python.exe
cachedir: .pytest_cache
rootdir: A:\AAA_algo_HW\tests
plugins: anyio-3.6.1, Faker-19.12.0, cov-4.1.0
collected 6 items

issue4.py::test_fit_transform_not_none PASSED                                                                                                                                                                  [ 16%]
issue4.py::test_fit_transform_type PASSED                                                                                                                                                                      [ 33%]
issue4.py::test_fit_transform_single_category PASSED                                                                                                                                                           [ 50%]
issue4.py::test_fit_transform_duplicate_categories PASSED                                                                                                                                                      [ 66%]
issue4.py::test_fit_transform PASSED                                                                                                                                                                           [ 83%]
issue4.py::test_fit_transform_no_categories PASSED                                                                                                                                                             [100%]

================================================================================================= 6 passed in 0.14s =================================================================================================